### ISSUE TITLE
TypeError: Cannot read property 'getBase64String' of undefined

### ISSUE THREAD

**Issue by Twinbird24** [Snapp-FidMe/react-native-img-to-base64#3]
I've followed the instructions in the README.md, but I'm having trouble using this library in my React Native project.  I'm using RN v. 0.49.3. When I import this library `import ImgToBase64 from 'react-native-image-base64';` and I take a look at `ImgToBase64` it is `undefined.`

I've tried uninstalling/ unlinking this library, reinstalling it and then restarting my iOS simulator and I'm still having this issue.

**Comment by Michaelvilleneuve** [comment]
That's weird. 
Open the xcode project and check that the lib is contained in that libraries folder of your xcode project.

If that's not the case, please try manually linking the lib.

The fact that you have `undefined` makes me think that the package is incorrectly downloaded or installed by npm.
Make sure that your `node_modules/react-native-image-base64` exists and that its content is similar to this Github

**Comment by Twinbird24** [comment]
My node_modules/react-native-image-base64 exists and its content is similar to this Github. The `RNImgToBase64.xcodeproj` file exists in the Libraries directory in Xcode.
<img width="214" alt="screen shot 2018-03-28 at 3 33 21 pm" src="https://user-images.githubusercontent.com/15930962/38051822-661d339e-329d-11e8-9376-47e8999f34d7.png">
I've linked the lib manually (as per the [RN docs](https://facebook.github.io/react-native/docs/linking-libraries-ios.html)) I've added the path to the library to the Header Search Paths as well in Xcode
<img width="508" alt="screen shot 2018-03-28 at 3 40 48 pm" src="https://user-images.githubusercontent.com/15930962/38052247-9da5b768-329e-11e8-8f0e-a031d17d0e4a.png">

I tried to launch my app again to test if it's working now, however, I got another (unrelated) error I'm trying to solve before I can check if the lib is working now :p https://stackoverflow.com/questions/49544015/react-native-error-ambiguous-resolution

I'll post back here once I'm able to get the iOS simulator to start.
